tests/mmdb.sh

```diff
@@ -2,13 +2,16 @@
 # added 2014-11-17 by singh.janmejay 
 # This file is part of the rsyslog project, released under ASL 2.0 
 echo =============================================================================== 
-echo \[mmdb.sh\]: test for mmdb 
-. $srcdir/diag.sh init 
-. $srcdir/diag.sh startup mmdb.conf 
-. $srcdir/diag.sh tcpflood -m 1 -j '202.106.0.20'
+echo \[mmdb.sh\]: test for mmdb
+# uncomment for debugging support:
+#export RSYSLOG_DEBUG="debug nostdout"
+#export RSYSLOG_DEBUGLOG="log"
+. $srcdir/diag.sh init
+. $srcdir/diag.sh startup mmdb.conf
+. $srcdir/diag.sh tcpflood -m 1 -j "202.106.0.20\ "
 echo doing shutdown 
-. $srcdir/diag.sh shutdown-when-empty 
+. $srcdir/diag.sh shutdown-when-empty
 echo wait on shutdown 
-. $srcdir/diag.sh wait-shutdown  
-. $srcdir/diag.sh content-check '"city" : "Beijing"' 
+. $srcdir/diag.sh wait-shutdown
+. $srcdir/diag.sh content-check '{ "city": "Beijing" }'
```

Comment:
can we be sure this IP will always remain valid for "Beijing"? In other words: is there a dedicated testing entry inside maxmindb? If so, we should use it. My fear is that in the distant future, this test might fail simply because the address assignment has changed. If we can't do it differently, we should put some comments into the test that tell us how we can find a new testing address. So that someone without knowledge of maxmindb has a chance to update the test.